src/Directory.Packages.props

```diff
@@ -1,49 +1,48 @@
+    <PackageVersion Include="Serilog.Sinks.File" Version="5.0.0" />
+    <PackageVersion Include="SonarAnalyzer.CSharp" Version="9.25.1.91650" />
+    <PackageVersion Include="JsonSchema.Net" Version="7.0.2" />
+    <PackageVersion Include="JsonPath.Net" Version="1.1.0" />
   </ItemGroup>
-
   <!-- Testing Packages -->
-	<ItemGroup>
-		<PackageVersion Include="Autofac" Version="8.0.0" />
-		<PackageVersion Include="Autofac.Extras.FakeItEasy" Version="7.0.0" />
-		<PackageVersion Include="coverlet.collector" Version="6.0.0" />
-		<PackageVersion Include="coverlet.msbuild" Version="6.0.2"/>
-		<PackageVersion Include="FakeItEasy" Version="8.2.0" />
-		<PackageVersion Include="FluentAssertions" Version="6.12.0" />
-		<PackageVersion Include="ImpromptuInterface" Version="8.0.4" />
-		<PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.1" />
-		<PackageVersion Include="Microsoft.Extensions.Configuration" Version="8.0.0" />
-		<PackageVersion Include="Microsoft.Extensions.Configuration.Abstractions" Version="8.0.0" />
-		<PackageVersion Include="Microsoft.Extensions.Configuration.Json" Version="8.0.0" />
-		<PackageVersion Include="Microsoft.Playwright.NUnit" Version="1.44.0" />
-		<PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
-		<PackageVersion Include="NUnit" Version="4.1.0" />
-		<PackageVersion Include="NUnit3TestAdapter" Version="4.5.0" />
-		<PackageVersion Include="NUnit.Analyzers" Version="4.2.0" />
-		<PackageVersion Include="Reqnroll" Version="1.0.1" />
-		<PackageVersion Include="Reqnroll.NUnit" Version="1.0.1" />
-		<PackageVersion Include="Testcontainers" Version="3.8.0" />
-	</ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <PackageVersion Include="Autofac" Version="8.0.0" />
+    <PackageVersion Include="Autofac.Extras.FakeItEasy" Version="7.0.0" />
+    <PackageVersion Include="coverlet.collector" Version="6.0.2" />
+    <PackageVersion Include="coverlet.msbuild" Version="6.0.2" />
+    <PackageVersion Include="FakeItEasy" Version="8.2.0" />
+    <PackageVersion Include="FluentAssertions" Version="6.12.0" />
+    <PackageVersion Include="ImpromptuInterface" Version="8.0.4" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.6" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration.Abstractions" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration.Json" Version="8.0.0" />
+    <PackageVersion Include="Microsoft.Playwright.NUnit" Version="1.44.0" />
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+    <PackageVersion Include="NUnit" Version="4.1.0" />
+    <PackageVersion Include="NUnit3TestAdapter" Version="4.5.0" />
+    <PackageVersion Include="NUnit.Analyzers" Version="4.2.0" />
+    <PackageVersion Include="Reqnroll" Version="2.0.1" />
```

Comment:
Thanks for applying this update(Reqnroll / Reqnroll.NUnit). We discussed it during the standup